src/main/java/org/folio/circulation/domain/policy/lostitem/LostItemPolicy.java

```diff
@@ -53,20 +56,27 @@ public static LostItemPolicy from(JsonObject lostItemPolicy) {
       getActualCostFee(lostItemPolicy),
       getFeeRefundInterval(lostItemPolicy),
       getBooleanProperty(lostItemPolicy, "returnedLostItemProcessingFee"),
-      getChargeOverdueFineProperty(lostItemPolicy)
+      getChargeOverdueFineProperty(lostItemPolicy),
+      getAgedToLostAfterOverdueIntervalProperty(lostItemPolicy)
     );
   }
 
+  private static Period getAgedToLostAfterOverdueIntervalProperty(JsonObject policy) {
+    final JsonObject intervalJson = policy.getJsonObject("itemAgedLostOverdue");
+
+    return intervalJson != null ? Period.from(intervalJson) : null;
+  }
+
   private static boolean getChargeOverdueFineProperty(JsonObject lostItemPolicy) {
     final String lostItemReturned = lostItemPolicy.getString("lostItemReturned");
     return "charge".equalsIgnoreCase(lostItemReturned);
   }
 
-  private static TimePeriod getFeeRefundInterval(JsonObject policy) {
+  private static Period getFeeRefundInterval(JsonObject policy) {
```

Comment:
Thank you for switching the code over to a single period class. At some point it would be good to remove the other when all usages are removed.